sdks/nodejs/src/agonesSDK.js

```diff
@@ -19,11 +19,30 @@ const services = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
-		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
+		this.client = new services.SDKClient('localhost:'+this.port(), grpc.credentials.createInsecure());
 		this.healthStream = undefined;
 		this.emitters = [];
 	}
 
+	port() {
```

Comment:
Minor but I tend to use a name like `getPort()` for this or you could convert the method to a getter `get port()` which would then be used as `this.port`. The current name implies to me that an action called `port()` is being performed.

Reply:
Thanks for the advice. I've changed it to a getter and updated the tests as well.